services/web-app/components/page-header/page-header.module.scss

```
@@ -34,24 +41,24 @@ $background-header: #d0e2ff;
 
   &-last {
     justify-content: flex-end;
-    @include breakpoint-down(md) {
+    @include breakpoint.breakpoint-down(md) {
       display: none;
     }
   }
 }
 
 .title {
-  margin-bottom: $spacing-07;
-  color: $text-primary;
-  @include type-style('display-01', true);
-  @include line-clamp(2);
-  @include breakpoint-down(md) {
-    @include type-style('quotation-02');
+  margin-bottom: spacing.$spacing-07;
+  color: theme.$text-primary;
+  @include type.type-style('display-01', true);
+  @include mixins.line-clamp(2);
+  @include breakpoint.breakpoint-down(md) {
+    @include type.type-style('quotation-02');
 
     font-family: font-family('sans');
```

```suggestion
    font-family: type.font-family('sans');
```

@jdharvey-ibm this should fix it.